### PR TITLE
Added ability to use multiple devices with the dashboard.

### PR DESCRIPTION
**Pull request by taraaliz**
This allows you to hook up multiple devices to the dashboard and control all of their statuses. I have also cleaned up a lot of the code, and made it so that only one file should be edited when adding new functions (demo.py).

### CODE REVIEW

File: dashboard.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import avahi
+
+from flask import Flask, request, render_template, make_response, current_app, json, Response, jsonify, redirect, url_for
+
+app = Flask(__name__)
+
+def get_device_list():
+	'''Returns a list of ips of devices'''
+	device_list = []
+	avahi.run()
```
Comment (elucas) [inline dashboard.py:11]: Overall, this commit is awesome, but I think we might want to refactor this bit.

This is running the avahi browser on every homepage request, which is a bit inefficient? Ideally I think we'd run the Avahi call from Cron every minute or two, and then read the BLINKT_DEVICES file created by it, in order to get _recent-ish_ status.

File: README.md [hunk 2]
```diff
@@ -38,14 +38,33 @@ sudo crontab -e
...
-+ This is not the finish product so there will be bugs!
++ This is not the finished product so there will be bugs!
 
 
 ### Getting Started!
-1. Run webpage.py & blinkt_status_display.py
-2. Visit [pi_local_ip_address]:5000 in any browser to edit your status!
-3. You can use any password for the first time you login (but make sure you remember it! - otherwise clear the blinkt_hash file and it will reset)
+1. Run dashboard.py
+2. Visit localhost:5000 in any browser to edit your status! If they are reported as missing, make sure they are powered up.
```
Comment (elucas) [inline README.md:46]: I think the dashboard runs on port 5001 by default

Comment (taraaliz) [reply README.md:46]: Correct, my bad

File: dashboard.py (hunk 1 above)
Comment (taraaliz) [reply dashboard.py:11]: Good idea, okay. You can delete that line and then add avahi.py to the setting up cron instructions in the readme then

**Pull request merged**